data/src/main/java/com/teamagam/gimelgimel/data/message/poller/MessageLongPoller.java

```diff
@@ -44,18 +40,20 @@ public MessageLongPoller(GGMessagingAPI messagingAPI, MessageDataMapper messageD
   public Observable poll() {
     long synchronizedDateMs = mPrefs.getLong(Constants.LATEST_MESSAGE_DATE_KEY);
 
-    return poll(synchronizedDateMs).map(this::getMaximumDate)
+    return poll(synchronizedDateMs)
+        .toFlowable(BackpressureStrategy.BUFFER)
```

Comment:
I don't agree, before moving to rx2 The `GGMessaingAPI` returned a simple `Observable` and only by the end of THIS method you used the backpressure operator (which returns a `Flowable`-like `Observable`). So IMO I can still use the `toFlowable` operator, but I agree that in the rest of the processing it should remain a `Flowable`.